packages/components/package.json

```diff
@@ -28,6 +28,7 @@
   "module": "dist/web3-ui-components.esm.js",
   "types": "dist/web3-ui-components.cjs.d.ts",
   "dependencies": {
+    "@web3-ui/hooks": "0.4.0",
```

Comment:
One more thing that came to my mind recently is that the `components` package should not rely on the `hooks` package in any way. We want both of these packages to function well together but also independently. 

This is why I feel like we shouldn't do any data fetching at all inside any of our `components`. Here is what I propose:

1. Move the current implementation to the `core` package.
2. Make the current implementation (inside `components`) so that it does not handle any data fetching but just accepts the data it needs to render.

Let me know if this is unclear. More than happy to discuss this in further detail since I think this is an important topic.

Reply:
Do you mean this component shouldn't have any state at all and just renders given props ?  Then we can remove the useTokenBalance hook from the Component. 
Is that what you meant in the second point?

Reply:
@AlexNi245 yes! The current verison of this component which _does_ have some state and fetches data should be in the `core` package and not in the `components` package. This is what we did recently for #121 too.

tldr:

- `<TokenBalance />` in `components` shouldn't have any state, it should just render some data that you pass to it.
- `<TokenBalance />` in `core` **should** have state and should use the `useTokenBalance` hook